src/audio/audio.cpp

```diff
@@ -36,6 +37,47 @@
 #endif
 
 /**
+@internal
+
+@class Audio::Private
+
+@brief Encapsulates private audio framework from public qTox Audio API.
+*/
+class Audio::Private
+{
+public:
+    Private()
+        : minInputGain{-30.0}
+        , maxInputGain{30.0}
+        , gain{0.0}
+        , gainFactor{0.0}
+    {
+    }
+
+    qreal inputGain() const {
+        return gain;
+    }
+
+    qreal inputGainFactor() const {
+        return gainFactor;
+    }
+
+    void setInputGain(qreal dB)
+    {
+        gain = qBound(minInputGain, dB, maxInputGain);
+        gainFactor = qPow(10.0, (gain / 20.0));
+    }
+
+public:
+    qreal   minInputGain;
+    qreal   maxInputGain;
+
+private:
```

Comment:
Look, a private private -> neat. :) Now values can only be changed through `setGain()`, making it "fool proof".